x/tokenfactory/keeper/keeper.go

```diff
@@ -83,6 +82,5 @@ func (k *Keeper) SetContractKeeper(contractKeeper types.ContractKeeper) {
 // it purely mints and burns them on behalf of the admin of respective denoms,
 // and sends to the relevant address.
 func (k Keeper) CreateModuleAccount(ctx sdk.Context) {
-	moduleAcc := authtypes.NewEmptyModuleAccount(types.ModuleName, authtypes.Minter, authtypes.Burner)
-	k.accountKeeper.SetModuleAccount(ctx, moduleAcc)
+	k.accountKeeper.GetModuleAccount(ctx, types.ModuleName)
```

Comment:
Does app wiring create module account by default? 🤔

Reply:
No, it should be created inside the genesis phase, see `x/gov` calls `GetModuleAccount` to create the module account as follows:
https://github.com/cosmos/cosmos-sdk/blob/release/v0.45.x/x/gov/genesis.go#L19